src/backend/distributed/commands/utility_hook.c

```diff
@@ -282,27 +287,50 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 		if (IsA(parsetree, DropStmt))
 		{
 			DropStmt *dropStatement = (DropStmt *) parsetree;
-			if (dropStatement->removeType == OBJECT_INDEX)
+			switch (dropStatement->removeType)
 			{
-				ddlJobs = PlanDropIndexStmt(dropStatement, queryString);
-			}
+				case OBJECT_INDEX:
+				{
+					ddlJobs = PlanDropIndexStmt(dropStatement, queryString);
+					break;
+				}
 
-			if (dropStatement->removeType == OBJECT_TABLE)
-			{
-				ProcessDropTableStmt(dropStatement);
-			}
+				case OBJECT_TABLE:
+				{
+					ProcessDropTableStmt(dropStatement);
+					break;
+				}
 
-			if (dropStatement->removeType == OBJECT_SCHEMA)
-			{
-				ProcessDropSchemaStmt(dropStatement);
-			}
+				case OBJECT_SCHEMA:
+				{
+					ProcessDropSchemaStmt(dropStatement);
+					break;
+				}
 
-			if (dropStatement->removeType == OBJECT_POLICY)
-			{
-				ddlJobs = PlanDropPolicyStmt(dropStatement, queryString);
+				case OBJECT_POLICY:
+				{
+					ddlJobs = PlanDropPolicyStmt(dropStatement, queryString);
+					break;
+				}
+
+				case OBJECT_TYPE:
+				{
+					ddlJobs = PlanDropTypeStmt(dropStatement, queryString);
```

Comment:
We are missing a break statement here.

(I did not realize this when I added a new case for functions after `OBJECT_TYPE` and broke things)

Reply:
Thanks, missed that one